web/tasks/Wiki-Auto.md

```diff
@@ -1,119 +1,213 @@
 
 #### Who are the source language producers?
 
-[More Information Needed]
+The dataset uses language from Wikipedia: some demographic information is provided [here](https://en.wikipedia.org/wiki/Wikipedia:Who_writes_Wikipedia%3F).
 
 ### Annotations
 
 #### Annotation process
 
-[More Information Needed]
+Sentence alignment labels were crowdsourced for 500 randomly sampled document pairs (10,123 sentence pairs total). The authors pre-selected several alignment candidates from English Wikipedia for each Simple Wikipedia sentence based on various similarity metrics, then asked the crowd-workers to annotate these pairs. Finally, they trained their alignment model on this manually annotated dataset to obtain automatically aligned sentences (138,095 document pairs, 488,332 sentence pairs).
 
 #### Who are the annotators?
 
-[More Information Needed]
+No demographic annotation is provided for the crowd workers.
 
 ### Personal and Sensitive Information
 
-[More Information Needed]
+Since the dataset is created from Wikipedia/Simple Wikipedia, all the information contained in the dataset is already in the public domain.
 
 ## Changes to the Original Dataset for GEM
 
-[More Information Needed]
+No change is made to the original dataset.
 
 ## Considerations for Using the Data
 
 ### Social Impact of the Dataset
 
-[More Information Needed]
+The dataset helps move forward the research towards text simplification by creating a larger and more accurate dataset. Progress in text simplification in turn has the potential to increase the accessibility of written documents to wider audiences.
 
 ### Impact on Underserved Communities
 
-[More Information Needed]
+The dataset is in English, a language with an abundance of existing resources.
 
 ### Discussion of Biases
 
-[More Information Needed]
+The dataset may contain some social biases, as the input sentences are based on Wikipedia. Studies have shown that the English Wikipedia contains both gender biases [(Schmahl et al., 2020)](https://research.tudelft.nl/en/publications/is-wikipedia-succeeding-in-reducing-gender-bias-assessing-changes) and racial biases [(Adams et al., 2019)](https://journals.sagepub.com/doi/pdf/10.1177/2378023118823946).
 
 ### Other Known Limitations
 
-[More Information Needed]
+Since the data is created using an automatic alignment model (which is not perfect) there could be still some alignment issues in the data.
 
 ## Getting started with in-depth research on the task
 
-[More Information Needed]
+The dataset can be downloaded from the original repository [(here)](https://github.com/chaojiang06/wiki-auto) by the authors (it also contains instructions to load a trained model on the dataset) or can also be used via HuggingFace and TFDS.
```

Comment:
Can you add links to the HF / TFDS mentions here? 

And maybe link papers that describe common approaches that should be used as baselines for people interested in working on the task.

Reply:
Added link to the HF and TFDS (for TFDS the PR is not merged yet, therefore added the link to the tfds homepage. Will update once it is merged). Also added the recent models proposed for simplification.